frontend/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import { Web3Provider } from './context/Web3Context'
 import { client } from './queries/client'
 import { WalletContextProvider } from './context/WalletContext'
 import { StatsContextProvider } from './context/StatsContext'
+import { UserInputContextProvider } from './context/UserInputContext'
```

Comment:
question: generally we shouldn't need relative imports, react-app is smart enough to figure out the imports relative to the root dir, is your linter throwing errors for this?

Reply:
ah yea, it's become sometimes i work in the root directory of the project (i.e. above `frontend/`)

Reply:
we can remove the relative imports during cleanup

Reply:
gotcha! sounds good!